### PR TITLE
Remove stretch reference from the repo

### PR DESCRIPTION
**Pull request by qianz2**
Removing stretch EOL reference from the runtime images.

- .NET 1.0, 1.1, 2.0, 2.1, 2.2
- Node 4, 6, 8, 9, 10
- PHP 5.6, 7.0, 7.2, 7.3
- Python 2.7, 3.6

- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [x] Tests are included and/or updated for code changes.
- [x] Proper license headers are included in each file.

### CODE REVIEW

File: images/runtime/dotnetcore/generateDockerfiles.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ declare -r DOCKERFILE_TEMPLATE="$DIR/template.Dockerfile"
 declare -r RUNTIME_BASE_IMAGE_NAME_PLACEHOLDER="%RUNTIME_BASE_IMAGE_NAME%"
 declare -r DOTNET_VERSION_PLACEHOLDER="%DOTNET_VERSION%"
 declare -r NETCORE_BUSTER_VERSION_ARRAY=($NET_CORE_APP_30 $NET_CORE_APP_50 $NET_CORE_APP_60 $NET_CORE_APP_70)
-declare -r NETCORE_STRETCH_VERSION_ARRAY=($NET_CORE_APP_10 $NET_CORE_APP_11 $NET_CORE_APP_20 $NET_CORE_APP_21 $NET_CORE_APP_22)
+declare -r NETCORE_STRETCH_VERSION_ARRAY=()
```
Comment (william-msft) [inline images/runtime/dotnetcore/generateDockerfiles.sh:21]: Do we still need the empty array?

File: tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs [hunk 2]
```diff
@@ -23,7 +23,7 @@ public NodeAngularAppTest(ITestOutputHelper output, TestTempDirTestFixture fixtu
         public const int PortInContainer = 4200;
 
         // Official Node.js version that is supported by Angular CLI 6.0+ is 8.9 or greater
-        [Theory]
+        [Theory(Skip = "Legacy node versions are out of support")]
```
Comment (william-msft) [inline tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs:26]: Do we plan on un-skipping? If not, should we remove the legacy tests altogether?

Comment (qianz2) [reply tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs:26]: I don't think there's any chance we'll unskip these tests in the future, so yes I can remove these tests.

File: images/runtime/dotnetcore/generateDockerfiles.sh (hunk 1 above)
Comment (qianz2) [reply images/runtime/dotnetcore/generateDockerfiles.sh]: Thanks, updated

File: vsts/pipelines/baseImages/php-fpm.yml [hunk 3]
```diff
@@ -34,8 +25,8 @@ jobs:
 
 - job: Release_PhpRuntimeBaseImage
   dependsOn: 
-    - Build_Php_Fpm_Base
     - Build_Php_Fpm_Buster_Base
+    - Build_PHP_FPM_Bullseye_Base
```
Comment (cormacpayne) [inline vsts/pipelines/baseImages/php-fpm.yml:29]: nit: can we change `FPM` to `Fpm`, or vice versa, just for consistency in this file 🙂

**Review by cormacpayne — APPROVED** [review]
Minor comments, otherwise LGTM -- excited to see the test time without the bloat 🙂

**Review by william-msft — APPROVED** [review]

File: images/runtime/node/10.1/base.stretch.Dockerfile [hunk 4]
```diff
@@ -1,29 +0,0 @@
-FROM oryxdevmcr.azurecr.io/private/oryx/oryx-node-run-base-stretch
```
[the remark this answers is not in the document]
Comment (qianz2) [reply images/runtime/node/10.1/base.stretch.Dockerfile:1]: Thanks for catching, I updated the `generateDockerfiles.sh` and removed the building section of node stretch images.

File: vsts/pipelines/baseImages/php-fpm.yml (hunk 3 above)
Comment (qianz2) [reply vsts/pipelines/baseImages/php-fpm.yml]: Updated

**Review by cormacpayne — APPROVED** [review]
The validation pipeline is speedy now 🥳

**Pull request merged**